canvas_modules/common-canvas/src/notification-panel/notification-panel.scss

```diff
@@ -47,49 +49,89 @@ $notification-left-border-color-width: 4px;
 	stroke: $ui-03;
 	position: absolute;
 	right: 16px;
+	top: 0;
 	width: 16px;
 }
 
 .notification-panel-header {
-	position: absolute;
-	height: $notification-panel-header-height;
+	min-height: $notification-panel-header-height;
 	margin-top: $notification-panel-arrow-height;
 	width: inherit;
-	background-color: $ui-02;
+	background-color: $ui-01;
 	display: flex;
 	flex-direction: column;
 	justify-content: center;
-	padding-left: 12px;
 	border: 1px solid $ui-03;
 	border-bottom: 0;
 	font-weight: 400;
-	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+	font-size: 16px;
+	padding: $spacing-03 $spacing-05;
+	.notification-panel-subtitle {
+		padding-top: $spacing-03;
+		font-size: 11px;
+		color: $text-02;
+	}
+	.notification-panel-close-icon {
+		position: absolute;
+		right: $spacing-03;
+		top: $spacing-05;
+		cursor: pointer;
+	}
 }
 
-.notification-panel-messages {
-	position: absolute;
-	width: inherit;
-	margin-top: $notification-panel-arrow-height + $notification-panel-header-height;
-	max-height: calc((100% / 2) - #{$notification-panel-header-height});
-	overflow-y: auto;
+.notification-panel-clear-all-container {
+	width: 100%;
+	height: $notification-panel-clear-all-height;
 	background-color: $ui-01;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
 	border: 1px solid $ui-03;
```

Comment:
The border above the "Clear all" should be just 1px, its combined with the bottom border from messages right now
<img width="343" alt="image" src="https://user-images.githubusercontent.com/1628100/83695981-22821900-a5b0-11ea-9b54-25db873a93d6.png">

Reply:
fixed